tools/clang/test/CodeGenHLSL/quick-test/streamout_matrix_all_orientations.hlsl

```diff
@@ -0,0 +1,45 @@
+// RUN: %dxc -E main -T gs_6_0 %s | FileCheck %s
+
+// CHECK: call float @dx.op.loadInput.f32(i32 4, i32 0, i32 0, i8 0, i32 0)
```

Comment:
Pay attention to the third and fourth args here, this is what shows that we're doing row or column-major loads and stores.